### ISSUE TITLE
add pure export

### ISSUE THREAD

**Issue by towc** [echoes-xyz/mongoose-geojson-schema#28]
This package currently works by performing side effects. That definitely discourages its usage. I'm going to look for alternatives now, but I would have definitely used this package if there was a way to require/import it without modifying other stuff.

There's no need to break any existing code. My suggestion is to have a `types/` folder, with each separate type as a file, that doesn't modify anything, but rather just exports the type, so you can import all of them through `const geoJsonTypes = require('mongoose-geojson-schema/types')`, or only `Point` through `const Point = require('mongoose-geojson-schema/types/Point')`.

The way I would more commonly use it is `const { Point, Polygon } = require('mongoose-geojson-schema/types');`

I would be happy to write up a PR for this, I just want to make sure it wouldn't get rejected because of philosophical reasons

**Comment by markstos** [comment]
I agree with this, although it would be a breaking change. Right now simply loading the module modifies the global `mongoose` instance.
